packages/react-components/src/organisms/ResultList.tsx

```diff
@@ -66,10 +69,29 @@ const ResultList: React.FC<ResultListProps> = ({
         />
       )}
     </header>
-    {numberOfItems > 0 && <main css={mainStyles}>{children}</main>}
-    <section css={pageControlsStyles}>
-      <PageControls {...pageControlsProps} />
-    </section>
+    {numberOfItems > 0 ? (
+      <>
+        <main css={mainStyles}>{children}</main>
+        <section css={pageControlsStyles}>
+          <PageControls {...pageControlsProps} />
+        </section>
+      </>
+    ) : (
+      <section
+        css={{
+          textAlign: 'center',
+        }}
+      >
+        <Paragraph primary>
+          <strong>No matches found!</strong>
+          <br />
+          <span css={{ color: lead.rgb }}>
+            We're sorry, we couldn't find results to match your search. Please
```

Comment:
urgh, this could appear with `searchQuery === ''`. but if it's specced this way in the ticket :shrug:

Reply:
I clarified this with Beatriz and its been noted in the ticket to behave like this